pkg/components/formatter/formatter_basic.go

```diff
@@ -0,0 +1,334 @@
+package formatter
+
+import (
+	"bytes"
+	"html/template"
+
+	diff "github.com/yudai/gojsondiff"
+)
+
+// A BasicDiff holds the stateful values that are used when generating a basic
+// diff from JSON tokens.
+type BasicDiff struct {
```

Comment:
Why is this formatter returning html and not the change model? The basic diff api should retrurn this BasicDiff model, the html rendering could have been done in the frontend, what was the advantage to render this to html in the backend?